### PR TITLE
Create Camera attributes widgets and fix Video Feed

### PR DESCRIPTION
**Pull request by matidr**
Created the following widgets:
* ISO
* Shutter
* F Stop
* EV
* WB

And fixed the Video Feed.

### CODE REVIEW

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jim-mcandrew) [inline .idea/jarRepositories.xml:1]: Any reason to commit this file?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -14,6 +14,8 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
 
+    <uses-feature android:name="android.hardware.camera" />
```
Comment (jim-mcandrew) [inline app/src/main/AndroidManifest.xml:17]: Why were these added?

File: app/src/main/assets/dronelink-kernel.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+/*! For license information please see dronelink-kernel.js.LICENSE.txt */
```
Comment (jim-mcandrew) [inline app/src/main/assets/dronelink-kernel.js:1]: This file should never be committed.

File: settings.gradle [hunk 4]
```diff
@@ -1,5 +1,5 @@
 include ':app', ':dronelink_core', ':dronelink_core_ui', ':dronelink_dji', 'dronelink_dji_ui'
-project(':dronelink_core').projectDir = new File(settingsDir, '../../private/dronelink-core-android/dronelink_core')
-project(':dronelink_core_ui').projectDir = new File(settingsDir, '../dronelink-core-ui-android/dronelink_core_ui')
-project(':dronelink_dji').projectDir = new File(settingsDir, '../dronelink-dji-android/dronelink_dji')
-project(':dronelink_dji_ui').projectDir = new File(settingsDir, '../dronelink-dji-ui-android/dronelink_dji_ui')
\ No newline at end of file
+project(':dronelink_core').projectDir = new File('../dronelink-core-android/dronelink_core')
```
Comment (jim-mcandrew) [inline settings.gradle:2]: We should probably have our local directory structure match (core in private, everything else in public).

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (matidr) [reply .idea/jarRepositories.xml:1]: I add it to the .gitignore don't know why is still adding it

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (matidr) [reply app/src/main/AndroidManifest.xml:17]: I was following DJI guidances for the feed. I'm gonna remove it since I think it's not needed.

File: app/src/main/assets/dronelink-kernel.js (hunk 3 above)
Comment (matidr) [reply app/src/main/assets/dronelink-kernel.js:1]: I'm gonna add it to the .gitignore then

File: settings.gradle (hunk 4 above)
Comment (matidr) [reply settings.gradle:2]: Sure

Comment (matidr) [reply settings.gradle:2]: Done

File: app/src/main/assets/dronelink-kernel.js (hunk 3 above)
Comment (matidr) [reply app/src/main/assets/dronelink-kernel.js:1]: Done

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (matidr) [reply app/src/main/AndroidManifest.xml:17]: Done

File: .idea/jarRepositories.xml (hunk 1 above)
Comment (matidr) [reply .idea/jarRepositories.xml:1]: Done
